pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -123,6 +123,7 @@ class CollectionViewSet(
     authentication_classes = []
     permission_classes = []
     serializer_class = CollectionSerializer
+    filterset_class = CollectionFilter
```

Comment:
what motivated this change?

Reply:
I wanted to filter by name/namespace on my test

Reply:
That makes sense, but then this would be introducing a new feature in the z-stream. Also it needs a changelog when introduced too.